templates/person/show.html.twig

```diff
@@ -29,163 +24,179 @@
+                                <a href="{{ path('geonames_show', {'id': person.cityOfDeath.geonameid}) }}">
+                                    {{ person.cityOfDeath.name }}
+                                </a>
+                            {% endif %}
+                        </td>
+                    </tr>
+                    {% if person.relatedFirms|length %}
+                        <tr>
+                            <th data-toggle='popover' data-content='{{ 'person.fields.relatedFirms'|trans }}'>Related Firms</th>
+                            <td>
+                                {% for firm in person.relatedFirms %}
+                                <a href='{{ path('firm_show', {'id': firm.id}) }}'>
+                                    {{ firm }}
+                                    </a>{% if not loop.last %}<br/>{% endif %}
+                                {% endfor %}
+                            </td>
+                        </tr>
+                    {% endif %}
+                    <tr>
+                        <th data-toggle="popover" data-content="{{ 'person.fields.viafUrl'|trans }}">VIAF URI</th>
+                        <td>
+                            {% if person.viafUrl %}
+                                <a href="{{person.viafUrl }}">{{ person.viafUrl }}</a>
+                            {% endif %}
+                        </td>
+                    </tr>
+                    <tr>
+                        <th data-toggle="popover" data-content="{{ 'person.fields.wikipediaUrl'|trans }}">Wikipedia Entry</th>
+                        <td>
+                            {% if person.wikipediaUrl %}
+                                <a href="{{person.wikipediaUrl }}">{{ person.wikipediaUrl }}</a>
+                            {% endif %}
+                        </td>
+                    </tr>
+                    <tr data-toggle="popover" data-content="{{ 'person.fields.imageUrl'|trans }}">
+                        <th>Image URL</th>
+                        <td>
+                            {% if person.imageUrl %}
+                                <a href="{{person.imageUrl }}">{{ person.imageUrl }}</a>
+                            {% endif %}
+                        </td>
+                    </tr>
+                    <tr>
+                        <th data-toggle="popover"  data-content="{{ 'person.fields.notes'|trans }}">Notes</th>
+                        <td>{{ person.notes }}</td>
+                    </tr>
+                    {% if is_granted('ROLE_USER') %}
+                        <tr data-toggle="popover" data-content="{{ 'person.fields.finalCheck'|trans }}">
+                            <th>Verified</th>
+                            <td>{% if person.finalCheck %}Yes{% else %}No{% endif %}</td>
+                        </tr>
+                    {% endif %}
+                {% endblock %}
+            {% endembed %}
+            {#<div class="person-info-block">
```

Comment:
This commented section probably needs to be cleaned up.

Reply:
Cleaned (by which I mean: deleted :-)